arangod/Graph/PathType.h

```diff
@@ -27,7 +27,12 @@ namespace arangodb {
 namespace graph {
 
 struct PathType {
-  enum class Type { KShortestPaths = 0, KPaths = 1, AllShortestPaths = 2 };
+  enum class Type {
+    KShortestPaths = 0,
+    KPaths = 1,
+    AllShortestPaths = 2,
+    ShortestPath = 3
```

Comment:
Nit: according to our naming convention, all enum values should be `k...`.
This can be adjusted in devel only.